src/latexify/frontend.py

```diff
@@ -21,6 +22,7 @@ def get_latex(
     use_raw_function_name: bool = False,
     use_signature: bool = True,
     use_set_symbols: bool = False,
+    prefixes: set[str] = set(),
```

Comment:
Mutable objects can't be used as the default values ([pylint-W0102](https://vald-phoenix.github.io/pylint-errors/plerr/errors/basic/W0102.html)). Use None instead and manually check if the argument is a set or None.

The following example shows why it is dangerous:
```
>>> def f(a=set()):
...     a.add(len(a))
...     return a
... 
>>> f()
{0}
>>> f()
{0, 1}
>>> f()
{0, 1, 2}
```

```suggestion
    prefixes: set[str] | None = None,
```